desktop/apps/page/test/integration.coffee

```diff
@@ -1,22 +0,0 @@
-{ startServer, closeServer } = require '../../../test/helpers/integration'
-Browser = require 'zombie'
-
-xdescribe 'Static page', ->
-
-  before (done) -> startServer done
-
-  after -> closeServer()
-
-  xit 'shows the page contents', (done) ->
-    browser = new Browser
-    browser.visit 'http://localhost:5000/terms', ->
-      browser.wait ->
-        browser.html().should.containEql 'This <em>page</em> is awesome!'
-        done()
-
-  xit 'shows the jobs page contents', (done) ->
-    browser = new Browser
-    browser.visit 'http://localhost:5000/job/terms', ->
-      browser.wait ->
-        browser.html().should.containEql 'This <em>page</em> is awesome!'
-        done()
```

Comment:
I think the actual `desktop/test/helpers/integration` include can also be removed, as well as the other references to it (e.g., in `tag/`).